patterns/templates/jolla-hw-adaptation-@DEVICE@.yaml

```diff
@@ -68,8 +64,17 @@ Requires:
 - pattern:jolla-sailfish-cellular-apps
 - telepathy-ring
 
-# For devices where test_gps eventually gets a fix:
-#- geoclue-provider-hybris
+# For GPS and online location service
+- geoclue-provider-hybris
+- geoclue-provider-mlsdb
```

Comment:
maybe you could add that devices not willing to ship MLS DB should remove this line